lua/lualine/components/aerial.lua

```diff
@@ -53,15 +55,56 @@ local default_options = {
   exact = true,
 }
 
-local function color_icon(symbol_kind, icon, colored)
+function M:get_highlight(name)
+  if vim.fn.hlexists(name) ~= 1 then
+    return {}
+  end
+  local hl_test = vim.api.nvim_get_hl_by_name(name, true)
+  -- Linked to NONE
+  if hl_test[true] == 6 then
+    return {}
+  end
```

Comment:
This just means it's an empty dictionary. From `:help lua-special-tbl`:

> `{[vim.type_idx]=vim.types.dictionary}` is converted to an empty dictionary

And indeed, `vim.type_idx == true` and `vim.types.dictionary == 6`.

However, in this case I think we can eliminate this function entirely and just use 
```lua
require('lualine.utils.utils').extract_highlight_colors(group, 'fg')
```

Reply:
Ah that makes sense.  I had tried to use this function before writing my own and was unsuccessful, but I must've had some other issue that was causing it to quietly fail.